<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="InternalLink">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ExternalLink">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>241</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="JavaScript">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>341</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="Resources">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>301</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="InternalCnt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>850</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ExternalCnt">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>850</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="JavaScriptCnt">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>850</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ResourcesCnt">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>850</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
